{"name": "test_mul_006", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_file2.Test002 object at 0x000002161F49FC20>\n\n    @pytest.mark.group3\n    def test_mul_006(self):\n        a = 2\n        b = 2\n        mul = a * b\n        print(\"Mul of a * b =\" + str(mul))\n        if mul == 5:\n            assert True\n        else:\n>           assert False\nE           assert False\n\ntest_file2.py:25: AssertionError"}, "attachments": [{"name": "stdout", "source": "bf6fc3a6-8845-4e8d-b056-12e2b75b70ca-attachment.txt", "type": "text/plain"}], "start": 1726145408955, "stop": 1726145408955, "uuid": "7515dd3e-8b64-4136-9842-51d4cce40baa", "historyId": "cdb5b3e79acb6b5df0c56db44f1f660f", "testCaseId": "cdb5b3e79acb6b5df0c56db44f1f660f", "fullName": "test_file2.Test002#test_mul_006", "labels": [{"name": "tag", "value": "group3"}, {"name": "suite", "value": "test_file2"}, {"name": "subSuite", "value": "Test002"}, {"name": "host", "value": "DESKTOP-PD08TMV"}, {"name": "thread", "value": "11772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file2"}]}